pypit/settings/settings.lris_blue

```diff
@@ -118,7 +118,7 @@ keyword dispangle None                 # Tilt
 
 ### Fits properties
 fits timeunit mjd                   # The unit of keyword+time (s=seconds, m=minutes, h=hours, or any of the astropy Time formats)
-fits calwin 12.0                    # The window of time in hours to search for calibration frames for a science frame
+fits calwin  0                      # The window of time in hours to search for calibration frames for a science frame
```

Comment:
This turns off the calwin checking.  I might suggest that be
the default for all instruments..
I am adding documentation on calwin now.